src/consumer.ts

```diff
@@ -142,7 +143,36 @@ export class Consumer extends TypedEventEmitter {
       this.emit('aborted');
     }
 
-    this.emit('stopped');
+    if (options?.waitForInFlightMessagesMs > 0) {
```

Comment:
References to `options?.waitForInFlightMessagesMs` here should probably be changed to `this.waitForInFlightMessagesMs`, and then `waitForInFlightMessagesMs` should be in the constructor, where it could then be defaulted to `0` like the other options.

Reply:
Also, I'm not too sure on the name `waitForInFlightMessagesMs`, is something like `pollingCompleteWaitTimeMs` better?